src/BookingForm.js

```diff
@@ -0,0 +1,195 @@
+import React, { useState } from "react";
+import SearchResults from "./SearchResults.js";
+import TableRow from "./TableRow.js";
+function BookingForm({ handleNewBooking }) {
+  const [isSubmitClicked, setSubmitClicked] = useState(false);
+  const [newBookingInfo, setNewbookingInfo] = useState({
+    id: "0",
+    title: "",
+    firstName: "",
+    surname: "",
+    email: "",
+    roomId: "",
+    checkInDate: "",
+    checkOutDate: "",
+    items: []
```

Comment:
This `items` array looks like you're trying to reference the whole array of bookings - instead you can just deal with the whole array of bookings in the parent component.